dali/python_function/python_function.cc

```diff
@@ -12,21 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "python_function.h"
+#include "dali/python_function/python_function.h"
```

Comment:
I think that `dali/python_function` is a bit too generic location for this code. How about to move this to the folder where the actual PythonFunction operator lives?
Something more like `dali/pipeline/operators/` or any other place where we plan to store the plugin?
@JanuszL @mzient Any preference?

Reply:
+1

Reply:
Either `dali/pipeline/operators/python_function` or `dali/plugins/python_function`.

Reply:
I think dali/pipeline/operators/python_function sounds reasonable.